.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,7 +23,6 @@ Using a simulator where a physical device is unavailable is acceptable.
 
 * [ ] iOS 13
 * [ ] iOS 14
-
```

Comment:
Sorry for the unrelated change here, but noticed this when I opened the PR, it was causing weird formatting in the version list.